### ISSUE TITLE
Return to Matchmaking bug

### ISSUE THREAD

**Issue by NocturnePlay** [ValveSoftware/Dota2-Gameplay#16180]
Hi there!

It happened to me to search for a game, accept it and for some reason, one of the square turns red, which automatically returns on searching. This bug (sort of ), repeats on and on.  It seems that if I sign out of steam and rejoin, at the second-third attempt, the matchmaking function works normally, yet it happened to me, on a secondary account, that I got banned because of this and I don't want this to happen on my main. ( worth to be mentioned that I scanned all the game files and nothing seems to be corrupted or lost )

Is there any permanent fix? 
Does this happened to somebody else too?

Cheers!

**Comment by commentator24** [comment]
are you talking about the 10 squares when you found and accept match? 
when it turns red it means one of players declined to accept / queue again (with dotaplus)

**Comment by NocturnePlay** [comment]
Yeah, but this time, it's a continuous loop. If I accept 10 times, it will return for searching, 10 times.

**Comment by TobyFlendersonFromHR** [comment]
that looks, pretty decent

**Comment by commentator24** [comment]
you sure it's not one of your party members keep declining? if one of the first 5 squares is red.
Happened to me cos one guy thought it was funny to keep pressing accept 10x and ppl type whats happening

**Comment by NocturnePlay** [comment]
I wasn't in any party. More than that, I got permanently banned on a secondary account because of this, due to ,, not being able to accept ..
So yes, believe me that there should be too big of a coincidence .. this happens for days.
And let's exclude what square turns red.. usually it's random.. it could be the third, 8th, 9th, doesn't matter.
I have read something that either points to some game files, or directly to the servers.
